src/main/java/teammates/ui/controller/InstructorStudentListPageCourseData.java

```diff
@@ -0,0 +1,25 @@
+package teammates.ui.controller;
+
+import teammates.common.datatransfer.CourseAttributes;
+
+/**
+ * Serves as a datatransfer class between {@link InstructorStudentListPageAction}
+ * and {@link InstructorStudentListPageData}
+ */
+public class InstructorStudentListPageCourseData {
```

Comment:
Is there any precedent for putting this kind of class in this package?
I'm not too sure if we should be placing a class that's not directly related to the `ControllerServlet` here.